packages/editor-ui/src/plugins/codemirror/completions/utils.ts

```diff
@@ -198,3 +198,9 @@ export const withSectionHeader = (section: CompletionSection): CompletionSection
 	section.header = renderSectionHeader;
 	return section;
 };
+
+export const isCompletionSection = (
+	section: CompletionSection | string | undefined,
+): section is CompletionSection => {
+	return !!section && typeof section === 'object' && 'name' in section;
+};
```

Comment:
This can be simpler - maybe no need for a utility.

```suggestion
export const isCompletionSection = (
	section: CompletionSection | string | undefined,
): section is CompletionSection => {
	return typeof section === 'object';
};
```